.codegen/api.java.tmpl

```diff
@@ -95,7 +95,7 @@ public class {{.PascalName}}API {
         try {
             Thread.sleep((long) (sleep * 1000L + Math.random() * 1000));
         } catch (InterruptedException e) {
-            Thread.currentThread().interrupt();
+            throw new DatabricksException("Current thread was interrupted", e);
```

Comment:
You have to call `Thread.currentThread().interrupt()` to reset the interrupted bit so that callers will know that this thread was interrupted and can handle such a change.

Reply:
I thought that we are already throwing an exception which should be handled by the callers. But I think it won't harm to reset the flag. Just in case someone suppresses the exception, it will still flag the thread as interrupted.